pkg/network/master/subnets.go

```diff
@@ -87,6 +140,13 @@ func (master *OsdnMaster) addNode(nodeName string, nodeIP string, hsAnnotations
 	}
 
 	// Create new subnet
+	annotations := make(map[string]string)
+	for key, value := range hsAnnotations {
+		annotations[key] = value
+	}
+	if len(nodeUID) > 0 {
+		annotations["nodeUID"] = nodeUID
```

Comment:
We prefer namespacing of annotations and '-' is preferred over camel case. For example: `network.openshift.io/node-uid`
Define the annotation in "<origin-repo>/pkg/network/apis/network/plugin.go" and use the NodeUIDAnnotation constant here.